vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -97,6 +97,22 @@ def textual_container_groups
     h
   end
 
+  def textual_lives_on
+    lives_on_ems = @record.lives_on.try(:ext_management_system)
+    return nil if lives_on_ems.nil?
+    # TODO: handle the case where the node is a bare-metal
+    {
+      :label => "Underlying Host",
```

Comment:
I was trying to catch both VM and bare-metal cases. Anyway the label can be built intelligently (checking if it's a VM or a bare-metal). I'll just hard-code Virtual Machine for now as bare-metal still has to be implemented.